### ISSUE TITLE
Add. 14,695 | https://bl.vuexistapps.us/manuscript/561

### ISSUE THREAD

**Issue by ZiyanZhang13** [srophe/britishLibrary-data#155]
### Problems with Manuscript Overview

1. the type of writing is missing on the web version 
2. the date is wrong in the web version, it is "xith or xiith cent" on the physical copy but "xth or xith cent" on the web

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

The physical description is completely incorrect. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

**Comment by haeleekim** [comment]
manuscript contents also completely wrong, this looks like the manuscript before actually CCCXL/Add. 14,507. there are extra additions in the print copy that is missing from this version

so where is the content for Add. 14,695?
- manuscript 558 is also registered as Add. 14,695, but its split into three sections and none of them have the same content as the print copy's Add. 14,695
- tried searching on website using keywords from the print and nothing is popping up

manuscript 201 is registered as Add. 14,507, and it has the correct info (so basically same contents as manuscript 561)

**Comment by wlpotter** [comment]
Thanks @haeleekim, yes we look to be missing a record for 14,695...Let me take a look at how we assigned these URIs in the decoder. It may be we will have you create a new record for 14,695.

So it looks like we have 14,507 as ms 201 (with the correct and complete encoding?)

ms 558 should be (according to the decoder) for 14,694, so maybe just a typo in the shelfmark? (see also the issues that are open https://github.com/srophe/britishLibrary-data/issues?q=is%3Aissue%20state%3Aopen%2014%2C694)

It looks like ms 561 just got incorrectly created, then, and we need to update it to the correct contents. @haeleekim let me know if you'd be interested in trying to update 561.xml to be for 14,695 (I don't notice anything too complicated when glancing at it).

Note to self: Cf. #1712 which is also about this problem.

**Comment by haeleekim** [comment]
Hi @wlpotter, sure I'd be down to update the contents. Do I just need to look at Wright's print and move everything there into  561.xml?

**Comment by wlpotter** [comment]
Yes, it will just be updating the various parts of the record based on the actual entry 561 should be for (Add. 14,695, entry CCCXLI). You should be able to just update the current 561.xml file rather than start from scratch, though if it's confusing to do it that way I can give you a blank template to work from instead

We do have Word Document copies of the Catalogue, here: https://github.com/srophe/wright-catalogue/tree/master/Wright%20Word%20Documents. So you can use that to copy/paste the text of the Catalogue -- though you'll want to proofread since there could be errors compared to the PDF.

Feel free to take a stab at it, and let me know if you run into any problems. If there are parts of the entry that you aren't sure how to encode, you can also just note them in this issue and I'll take a look at it
